code/modules/projectiles/projectile/beams.dm

```diff
@@ -203,17 +203,23 @@ In pvp they also have more lasting damages, such as infections, and pain form bu
 		L.adjustBruteLoss(-3)
 		L.adjustFireLoss(-3)
 
-/obj/item/projectile/beam/sniper/healing/staff
+/obj/item/projectile/beam/sniper/harmony
 	name = "harmony"
+	icon_state = "xray"
+	damage_types = list(TOX = 0)//Shouldnt do anything but just in case its toxin
+	armor_penetration = 0
+	stun = 0
+	weaken = 0
+	eyeblur = 0
+	muzzle_type = /obj/effect/projectile/laser_blue/muzzle
+	tracer_type = /obj/effect/projectile/laser_blue/tracer
+	impact_type = /obj/effect/projectile/laser_blue/impact
 
-/obj/item/projectile/beam/sniper/healing/staff/on_hit(atom/target, blocked = FALSE)
-	. = ..()
+/obj/item/projectile/beam/sniper/harmony/on_hit(atom/target, blocked = FALSE)
```

Comment:
Dont  make it inject chemicals. The way it was was done in a direct way for a direct reason